js/plugins/abbr/plugin.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright (c) 2014-2022, CKSource Holding sp. z o.o. All rights reserved.
+ * Licensed under the terms of the MIT License (see LICENSE.md).
+ *
+ * Basic sample plugin inserting abbreviation elements into the CKEditor editing area.
+ *
+ * Created out of the CKEditor Plugin SDK:
+ * https://ckeditor.com/docs/ckeditor4/latest/guide/plugin_sdk_sample_1.html
+ */
+
+// Register the plugin within the editor.
+CKEDITOR.plugins.add( 'abbr', {

```

Comment:
Can't this be added in the same way our other ckeditor plugins are? https://github.com/SU-SWS/stanford_text_editor/blob/8.x/composer.json#L7

Reply:
So that's a great question.  It's not an official plugin; it's actually taken from CKEditor's own documentation about how to write plugins.  Here's the original source: https://github.com/ckeditor/ckeditor4-docs-samples/tree/master/tutorial-abbr-2/abbr

As you can see, it's a couple layers down into the repo instead of standing alone.  I suppose we could pull it out and put it in a separate repo, but I'm not sure what we'd get in terms of benefit there.  Nobody else would be updating it, so it would never change.

Reply:
Ok that makes sense then... next question... do we need a button for it?

Reply:
because we'll have to re-build this for Ckeditor 5 later

Reply:
Well, the button was the ask in the ticket: https://stanfordits.atlassian.net/browse/D8CORE-1835

If we want to decide not to do it, that's fine too, but maybe we should ask @cjwest. 🤷

Reply:
I'd rather just allow the tag, but not do the whole button approach. It's not a very common element anyways. 
Sorry that you did the development to get the button.
Keep this PR open just in case we decide we want it later.

otherwise, lets adjust the profile PR to just add the tag as allowed.